### PR TITLE
Linux eonent issue 4

### PR DESCRIPTION
**Pull request by AGhost-7**
See #4

### CODE REVIEW

**Comment by jonschlinkert** [comment]
thank you! much appreciated

File: index.js [hunk 1]
```diff
@@ -9,8 +9,23 @@
...
+	},
+	linux: {
+		name: 'xclip',
+		args: ['-selection', 'clipboard']
+	}
+};
+
 function toClipboard(args, cb) {
-  var proc = cp.spawn(program(), {
+  var proc = cp.spawn(program.name, program.args, {
```
Comment (jonschlinkert) [inline index.js:28]: wait, how did this work for you locally? `program.name` doesn't exist. `process.platform` is used to get the value to use. also, you only updated the `sync` code to be a string.

Comment (AGhost-7) [inline index.js:28]: Seems like something got out of sync somehow. Its supposed to have `[process.platform]` right after the object definition. Gimme a sec.

**Comment by AGhost-7** [comment]
Looks like its not syncing my changes, should I just open a new PR?

**Pull request merged**